blockbase/sass/post/_meta.scss

```diff
@@ -1,4 +1,5 @@
 .post-meta {
+	row-gap: var(--wp--custom--gap--baseline) !important; // !important is needed to override the unique .wp-container-ID classes
```

Comment:
It would be good to link the GB issue that suggests different vertical and horizontal block gaps, so we can remove this when that gets fixed.

Reply:
Ah yeah good idea, will sort this now